src/components/Spotlight/Spotlight.stories.tsx

```diff
@@ -109,7 +109,137 @@ export const AnimatedSpotlight: Story = (args: Partial<SpotlightProps>) => {
         </p>
       </Card>
       {tourStarted && (
-        <Spotlight {...args} StyledAnnotation={Annotation} targetElement={stepOptions[currStep]}>
+        <Spotlight
+          {...args}
+          StyledAnnotation={Annotation}
+          targetElement={stepOptions[currStep]?.current as HTMLElement}
+        >
+          <Text color="white" containerProps={{ as: 'h1', style: { fontSize: '2em' } }}>
+            {messages[currStep].title}
+          </Text>
+          <Text color="white" containerProps={{ style: { fontWeight: '700' } }}>
+            {messages[currStep].subtitle}
+          </Text>
+          <br />
+          <br />
+          <Button
+            color={colors.background}
+            iconSuffix={mdiChevronDoubleRight}
+            variant={variants.outline}
+            onClick={() => {
+              setStep(0);
+              setTour(false);
+            }}
+          >
+            Skip
+          </Button>
+          &nbsp;
+          <Button
+            StyledContainer={NextButtonContainer}
+            iconSuffix={lastStep ? mdiCheckBold : mdiArrowRightBold}
+            elevation={1}
+            color={lastStep ? colors.secondaryDark : colors.primaryDark}
+            onClick={() => {
+              goNext();
+            }}
+          >
+            {lastStep ? 'I got it' : 'Next'}
+          </Button>
+        </Spotlight>
+      )}
+    </>
+  );
+};
+Default.args = {
+  padding: 12,
+  shape: 'rounded box',
+  animateTargetChanges: true,
+  backgroundDarkness: 0.3,
+  backgroundBlur: '0.25rem',
+  cornerRadius: 12,
+};
+
+export const CustomScrollWindow: Story = (args: Partial<SpotlightProps>) => {
+  const [currStep, setStep] = useState<number>(0);
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const cardRef = useRef<HTMLDivElement>(null);
+  const bodyRef = useRef<HTMLDivElement>(null);
+  const menuRef = useRef<HTMLButtonElement>(null);
+  const [tourStarted, setTour] = useState<boolean>(false);
+
+  const stepOptions = [null, menuRef, cardRef, buttonRef];
+  const messages = [
+    {
+      title: 'Welcome to the tutorial!',
+      subtitle: 'targetElement is null for this part of the tour, so nothing is highlighted!',
+    },
+    { title: 'This is a kebab menu.', subtitle: '' },
+    { title: 'This is the whole card', subtitle: '' },
```

Comment:
Some punctuation for consistency

```suggestion
    { title: 'This is the whole card.', subtitle: '' },
```